von/strparse.py

```diff
@@ -95,19 +191,26 @@ def toAOPS(text: str) -> str:
     text = re.sub(r"\\textit{([^}]*)}", r"[i]\1[/i]", text)
     text = re.sub(r"\\textbf{([^}]*)}", r"[b]\1[/b]", text)
     text = re.sub(
-        r"\\paragraph{([^}]*)}", DIVIDER + r"[color=blue][b]\1[/b][/color]", text
+        r"\\paragraph{([^}]*)}", DIVIDER + r"[color=blue][b]\1[/b][/solor]", text
     )
     text = re.sub(r"\\subparagraph{([^}]*)}", DIVIDER + r"[b]\1[/b]", text)
     text = re.sub(r"\\url{([^}]*)}", r"[url]\1[/url]", text)
     text = re.sub(r"\\href{([^}]*)}{([^}]*)}", r"[url=\1]\2[/url]", text)
     text = re.sub(
         r"\\item\[([^\]]*)\]", r"[*] [b]\1[/b]", text
     )  # for description items
-    text = text.replace(r"\arc", r"\widehat")
 
-    # Join together newlines
-    paragraphs = [
-        " ".join([line.strip() for line in paragraph.splitlines()]).strip()
-        for paragraph in text.split("\n\n")
-    ]
-    return "\n".join(paragraphs)
+    return text
+    # # Join together newlines
+    # paragraphs = [
+    #     " ".join([line.strip() for line in paragraph.splitlines()]).strip()
+    #     for paragraph in text.split("\n\n")
+    # ]
+    # return "\n".join(paragraphs)
+
+
+if __name__ == "__main__":
```

Comment:
i don't think in the public version we should have additional pyperclip dependency or this block